solr/solr-ref-guide/modules/deployment-guide/pages/monitoring-with-prometheus-and-grafana.adoc

```diff
@@ -587,3 +594,27 @@ This screenshot shows what it might look like:
 
 .Grafana Dashboard
 image::monitoring-with-prometheus-and-grafana/grafana-solr-dashboard.png[image,width=800]
+
+== Prometheus Metrics API
```

Comment:
Not sure we want to name it this way... perhaps "Metrics API with Prometheus format"